### ISSUE TITLE
[FEATURE] Handle golang blocks

### ISSUE THREAD

**Issue by benjamineskola** [Riverside-Healthcare/djLint#734]
- [x] I've searched the [issues](https://github.com/Riverside-Healthcare/djLint/issues)
- [x] I've read the [docs](https://djlint.com)

## Feature Request

Golang templates behave differently from e.g. Django/Jinja: instead of having variables `{{ variable | filter }}` and blocks `{% if foo %}bar{% endif %}`, both variables and blocks use the double-curly-bracket syntax: `{{ if foo }}{{ bar }}{{ end }}`.

As far as I can tell, djlint doesn't recognise this, so blocks in a golang template don't get any indentation: they're treated just like variables.

```html
{{ define "main" }}
{{ if .Page }}
{{ .Page.Title }}
{{ end }}
{{ end }}
```

```sh
$ djlint --reformat --indent 2 --profile golang test.html
```

The above has no impact. However if I change the "block" syntax to use Django-style `{% … %}`, then as expected the contents of the if statement are indented (even with `--profile golang`). I think the golang profile should recognise `{{ … }}` as a potential block syntax, depending on the contents.
